src/test/java/com/wcc/platform/service/CmsServiceTest.java

```diff
@@ -24,6 +24,10 @@
 import org.mockito.MockitoAnnotations;
 
 class CmsServiceTest {
+
+  public static final int DEFAULT_CURRENT_PAGE = 1;
+  public static final int DEFAULT_PAGE_SIZE = 10;
```

Comment:
also created here again, why are you not using the SetupFactories?

Reply:
I don't see this fixed.

Reply:
I think they were reintroduced when I was dealing with the merge conflicts. I have taken them all out and used the SetupFactories